woocommerce/wc-admin
# Home Screen: Update Store Management (Quick Links) icons

Fixes #4396

This PR updates the icons used in the Store Management (Quick Links) card to be those from `@wordpress/icons`, which is what was intended in the design.

It is a followup to #4350, where the Store Management card was introduced.

### Screenshots

<img width="1189" alt="ScreenCapture at Tue May 19 16:02:12 EDT 2020" src="https://user-images.githubusercontent.com/2098816/82372881-c386a680-99ea-11ea-9c4d-02c45e7c6fd9.png">

### Detailed test instructions:

- View the new home screen.
- Verify that the icons shown match those in the screenshot.

### Changelog Note:

No changelog note needed.
